src/Nix/Exec.hs

```diff
@@ -753,7 +755,11 @@ addTracing k v = do
             opts :: Options <- asks (view hasLens)
             let rendered =
                     if verbose opts >= Chatty
+#ifdef MIN_VERSION_pretty_show
```

Comment:
This was the minimal change I could find to get it building.

Reply:
OK, I'll give it a go later tonight and see what I can find, thanks.

Reply:
Hmm, it would seem that your'e not using the version of `hpack` that's specified in `hnix`s `default.nix`, since all the Travis builds are failing due to the `.cabal` being out of sync again.

Reply:
Oh, I probably forgot to re-run hpack.  I'll push an update.

Reply:
Ah, I didn't realize this was in `Exec.hs`, not `Main.hs`, so these changes are good.